schema-derivatives-options.graphql

```diff
@@ -317,9 +317,18 @@ type DerivOptProtocol implements Protocol @entity {
   " Total number of contracts exercised+expired "
   contractsClosedCount: Int!
 
+  " Notional value of all open positions "
+  openInterestUSD: BigDecimal!
+
+  " Cumulative historical value of all open positions "
+  cumulativeOpenInterestUSD: BigDecimal!
+
   " Total number of open positions (total positions open-closed) "
   openPositionCount: Int!
 
+  " Cumulative number of open positions (minted positions) "
+  cumulativeOpenPositionCount: Int!
```

Comment:
I feel like these don't make sense as cumulative fields, since they're both current values, also there is already `contractsTakenCount` and `cumulativeVolumeUSD` which should cover these?

Reply:
Remove them at 61c6e61809716052f8094511e3353644e12d6d89 Thanks!